pubspec.yaml

```
@@ -12,19 +12,20 @@ environment:
 
 dependencies:
   collection: ^1.17.1
+  copy_with_extension: ^6.0.1
+  equatable: ^2.0.5
```

```suggestion
  equatable: ^2.0.7
```
Let's use the latest version